lib/src/layer/scalebar/painter/base.dart

```diff
@@ -0,0 +1,5 @@
+part of '../scalebar.dart';
+
+abstract class ScalebarPainter extends CustomPainter {
```

Comment:
Why does this exist if there's only one extension and no possibility for users to create/use their own? Forward planning?

Reply:
Yes it's a bit of forward planning. The Scalebar widget needs to know from the painter on how large it should be.